app/models/study_file.rb

```diff
@@ -766,8 +766,13 @@ def parse_fail_bucket_location
 
   # generate a download path to use with bulk_download
   # takes the form of :study_accession/:output_directory_name/:filename
-  def bulk_download_pathname
-    "#{self.study.accession}/#{self.output_directory_name}/#{self.upload_file_name}"
+  # supports Unix- and Windows-formatted paths
+  def bulk_download_pathname(os: '')
+    path = "#{self.study.accession}/#{self.output_directory_name}/#{self.upload_file_name}"
+    if os =~ /Win/
+      path.gsub!(%r{/}, '\\') # reformat with backslashes once computed to include bulk_download_type
+    end
+    path
```

Comment:
feels like there might be some de-duplication possible between this and `bulk_download_pathname` in directory_listing.  Just a couple lines, so not too big of a deal though